sdk/eventgrid/azure-eventgrid/samples/consume_samples/functionsapp/.gitignore

```diff
@@ -0,0 +1,130 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
the functions app has a gitignore?  I guess weirder things have happened, surprised this doesn't duplicate functionality from .funcignore.  (Take this ramble as, "is this needed for the sample?")

Reply:
I don't think it's needed for the sample. I'm not even sure if the functionsapp should be in the sample.

Reply:
it looks like this .gitignore covers different things than .funcignore, so I'll keep both for now